src/new/wallet.js

```diff
@@ -156,6 +162,34 @@ class HathorWallet extends EventEmitter {
 
     // This object stores pre-processed data that helps speed up the return of getBalance and getTxHistory
     this.preProcessedData = {};
+
+    if (multisig) {
+      this.multisig = {
+        pubkeys: multisig.pubkeys,
+        minSignatures: multisig.minSignatures,
+      };
+    }
+  }
+
+  /**
+   * Gets the xpub derived at the MultiSig account path
+   *
+   * @param {Object} options Only used when deriving from seed, may include passphrase and networkName.
+   */
+  getMultisigPublicKey(options) {
+    if (this.isFromXPub()) {
+      throw new WalletFromXPubGuard('getMultisigPublicKey');
+    }
+
+    if (this.seed) {
+      return walletUtils.getMultiSigXPubFromWords(this.seed, options);
+    }
+
+    if (this.xpriv) {
+      return walletUtils.getMultiSigXPubFromXPriv(this.seed);
+    }
+
+    throw "This should never happen";
```

Comment:
Use single quotes

Actually, this will happen if `getMultisigPublicKey` is called after `clearSensitiveData` is called from the `start()` method

Reply:
This method was removed.
The idea was to generate the multisig pubkey, but the caller doesn't need to start the wallet to do so.
That's why I opted to use the `walletUtils` directly and not have this method on the `HathorWallet`.

Reply:
Looks good